src/libcore/num/mod.rs

```diff
@@ -140,6 +140,7 @@ macro_rules! int_impl {
         /// Leading and trailing whitespace represent an error.
         ///
         /// # Examples
+        /// Basic usage:
```

Comment:
Oh, sorry, can you put a space between this and the `# Examples`? This is cramped, I know it still works markdown-wise